Scripts/build-webdriveragent.js

```diff
@@ -37,52 +34,13 @@ async function buildWebDriverAgent (xcodeVersion) {
     throw e;
   }
 
-  // Create bundles folder
-  const pathToBundles = path.resolve(rootDir, 'bundles');
-  await mkdirp(pathToBundles);
-
-  // Start creating zip
-  const uncompressedDir = path.resolve(rootDir, 'uncompressed');
-  await fs.rimraf(uncompressedDir);
-  await mkdirp(uncompressedDir);
-  log.info('Creating zip');
-
-  // Move contents of the root to folder called "uncompressed"
-  await exec('rsync', [
-    '-av', '.', uncompressedDir,
-    '--exclude', 'node_modules',
-    '--exclude', 'build',
-    '--exclude', 'ci-jobs',
-    '--exclude', 'lib',
-    '--exclude', 'test',
-    '--exclude', 'bundles',
-    '--exclude', 'azure-templates',
-  ], {cwd: rootDir});
-
-  // Move DerivedData/WebDriverAgent-* from Library to "uncompressed" folder
-  const wdaPath = (await fs.glob(`${derivedDataPath}/WebDriverAgent-*`))[0];
-  await mkdirp(path.resolve(uncompressedDir, 'DerivedData'));
-  await fs.rename(wdaPath, path.resolve(uncompressedDir, 'DerivedData', 'WebDriverAgent'));
-
-  // Compress the "uncompressed" bundle as a Zip
-  const pathToZip = path.resolve(pathToBundles, `webdriveragent-xcode_${xcodeVersion}.zip`);
-  await zip.toArchive(
-    pathToZip, {cwd: path.join(rootDir, 'uncompressed')}
-  );
-  log.info(`Zip bundled at "${pathToZip}"`);
-
-  // Now just zip the .app and place it in the root directory
-  // This zip file will be published to NPM
-  const wdaAppBundle = 'WebDriverAgentRunner-Runner.app';
-  const appBundlePath = path.join(uncompressedDir, 'DerivedData', 'WebDriverAgent',
-    'Build', 'Products', 'Debug-iphonesimulator', wdaAppBundle);
-  const appBundleZipPath = path.join(rootDir, `${wdaAppBundle}.zip`);
+  const zipName = `WebDriverAgentRunner-Runner-Sim-${xcodeVersion}.zip`;
+  log.info(`Creating ${zipName} which includes ${wdaAppBundle}`);
+  const appBundleZipPath = path.join(rootDir, zipName);
   await fs.rimraf(appBundleZipPath);
-  log.info(`Created './${wdaAppBundle}.zip'`);
-  await zip.toArchive(appBundleZipPath, {cwd: appBundlePath});
+  log.info(`Created './${zipName}'`);
+  await zip.toArchive(appBundleZipPath, {pattern: '*.app/**', cwd: appBundlePath});
```

Comment:
I would say using system `zip` would be faster here as it supports multicore CPUs

Reply:
makes sense